api/v1alpha1/databasesecretenginerole_types.go

```diff
@@ -38,12 +38,12 @@ type DatabaseSecretEngineRoleSpec struct {
 
 	// DeafulTTL Specifies the TTL for the leases associated with this role. Accepts time suffixed strings ("1h") or an integer number of seconds. Defaults to system/engine default TTL time.
 	// +kubebuilder:validation:Optional
-	// +kubebuilder:default=0s
+	// +kubebuilder:default=0
```

Comment:
this was intentional, are you sure that it should be `0`.

Reply:
The generator was complaining about an unexpected 's'

Reply:
because it needs to be a string...so quotes